<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cliente</class>
 <widget class="QMainWindow" name="Cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cliente</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Aplicacion Cliente Resultados del Mundial</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lswSelecciones">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>231</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Qatar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ecuador</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Senegal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Paises Bajos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inglaterra</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Iran</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Estados Unidos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gales</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Argentina</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arabia Saudi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mexico</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Polonia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Francia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Australia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dinamarca</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tunez</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Espania</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Costa Rica</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Alemania</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Japon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Belgica</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Canada</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marruecos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Croacia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brasil</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Portugal</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblEquipos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipos del mundial de Qatar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPartidoDe">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Partidos de Qatar:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twdPartidos">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>581</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Equipo1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Equipo2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hora</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="lblPartidoDe_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jugadores de Qatar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lswJugadores">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>221</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>351</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Equipos</string>
    </property>
    <widget class="QLabel" name="lblFederacion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Federacion:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="letPuntos">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lblDirector">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Director Tecnico:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPartidosP">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Partidos Pendientes:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSeleccion">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Qatar</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblGolesC">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Goles en Contra:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="letAusencias">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="letGolesF">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>270</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="letPartidosJ">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lblGolesF">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Goles a Favor:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="letDirector">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="letPartidosP">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="letFederacion">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="letGolesC">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>300</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lblPuntos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Puntos:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblGrupo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Grupo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAusencia">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ausencias:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="letGrupo">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lblPartidosJ">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Partidos Jugados:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionConexion_Server"/>
    <addaction name="actionCarga_de_equipos"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConexion_Server">
   <property name="text">
    <string>Conexion Server</string>
   </property>
  </action>
  <action name="actionCarga_de_equipos">
   <property name="text">
    <string>Carga de equipos</string>
   </property>
  </action>
  <action name="actionIngreso_de_prediccion">
   <property name="text">
    <string>Ingreso de prediccion</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionConexion_Server</sender>
   <signal>triggered()</signal>
   <receiver>Cliente</receiver>
   <slot>Conectar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lswSelecciones</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>Cliente</receiver>
   <slot>Seleccionar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Conectar()</slot>
  <slot>Seleccionar()</slot>
 </slots>
</ui>
